python/spear/urdf/fetch.xml

```diff
@@ -522,20 +522,20 @@
             <inertia ixx="6.98014e-06" ixy="-2.62572e-08" ixz="-1.58206e-08" iyy="4.12665e-05" iyz="-1.57909e-09" izz="3.75763e-05"/>
         </inertial>
         <visual>
-            <origin rpy="0 0 0" xyz="0 -0.101425 0"/>
+            <origin rpy="0 0 0" xyz="0 0.101425 0"/>
```

Comment:
Do these changes need to be accompanied by corresponding changes to our `UrdfBot` plugin, so that our `open_loop_control_fetch` demo still behaves as expected?

Reply:
Let's assume: (1) the URDF-to-MJCF parsing code in this PR is correct; (2) Mujoco loads `output.xml` correctly; and (3) all the meshes in `urdf_meshes.zip` have been exported correctly.

Under these assumptions, we can use Mujoco's behavior when loading MJCF files as a reference for how SPEAR should behave. In other words, the Fetch robot spawned by the `UrdfBot` plugin in SPEAR should match the Fetch robot spawned in Mujoco. Ideally, we would match with Mujoco numerically and visually, but if we need to choose one or the other, we should match numerically.

If SPEAR does not match the behavior of Mujoco, we should fix our `UrdfBot` code in this PR, and also apply the coordinate system change above, because then we'll have exactly the same URDF behaving exactly the same way in SPEAR and Mujoco.